docs/how-to/hip_programming_guide.rst

```diff
@@ -0,0 +1,55 @@
+.. meta::
+    :description: HIP programming guide
+    :keywords: CU, CUs, number of CUs, compute units
+
+.. _hip-programming-guide:
+
+********************************************************************************
+HIP programming guide
+********************************************************************************
+
+The programming guide sections are the following:
```

Comment:
I thought we discussed a bit more in the Intro here? Or was this something I was supposed to write?

Reply:
I did not manage to add the text there yet.